src/coreclr/jit/fgflow.cpp

```diff
@@ -556,11 +552,11 @@ Compiler::SwitchUniqueSuccSet Compiler::GetDescriptorForSwitch(BasicBlock* switc
         {
             FlowEdge* const   succEdge = swtDesc->bbsDstTab[i];
             BasicBlock* const targ     = succEdge->getDestinationBlock();
-            if (BitVecOps::IsMember(&blockVecTraits, uniqueSuccBlocks, targ->bbNum))
+            if (BitVecOps::IsMember(&blockVecTraits, uniqueSuccBlocks, targ->bbID))
             {
                 nonDups[nonDupInd] = succEdge;
                 nonDupInd++;
-                BitVecOps::RemoveElemD(&blockVecTraits, uniqueSuccBlocks, targ->bbNum);
+                BitVecOps::RemoveElemD(&blockVecTraits, uniqueSuccBlocks, targ->bbID);
             }
```

Comment:
Ditto here, it seems unnecessary to change this.

Reply:
This switch to `bbID`-based `BitVecs` isn't Debug-only like the other ones, though the sparser set doesn't seem to have impacted [TP](https://dev.azure.com/dnceng-public/public/_build/results?buildId=875722&view=ms.vss-build-web.run-extensions-tab) all that much.

Reply:
Yeah, I would be surprised if the sparser bit sets are going to have any noticeable impact on performance.. A few bits wasted due to optimizing out entire basic blocks is going to be a drop in the bucket compared to everything else that's happening when handling the IR in those basic blocks.

My feedback here probably more comes down to bikeshedding over the naming of the final field we are going to end up with...